classes/cluster/mk20_stacklight_advanced/openstack/compute.yml

```diff
@@ -3,6 +3,7 @@ classes:
 - system.linux.system.repo.mos8
 - system.nova.compute.cluster
 - system.opencontrail.compute.cluster
+- service.opencontrail.compute.cluster
```

Comment:
@cznewt @simonpasquier unless I misunderstand something, this was what I did in my 1st proposed PR but as Simon suggested this would break other labs therefore the new solution